### ISSUE TITLE
501 Not Implemented when creating custom portal url in apache per instructions

### ISSUE THREAD

**Issue by czcpf** [espocrm/espocrm#2737]
**Describe the bug**
When following the instructions listed at https://docs.espocrm.com/administration/portal/apache-configuration/ for v7 for a custom portal, the calls to any script involving /api/v1/portal-access/ result in a 501 Not implemented error. 

**To Reproduce**
1. Fresh install of Espo 7.4.3
2. Create a custom portal with custom url
3. In apache .conf file, configure in accordance with documentation specified above:

```
DocumentRoot /{path_to_espo}/public/portal/
Alias /client/ /{path_to_espo}/client/
Alias /api/v1/ /{path_to_espo}/public/api/v1/

<Directory /{path_to_espo}/public/>
    AllowOverride All
</Directory>

<IfModule mod_rewrite.c>
    RewriteEngine On
    RewriteRule .* - [E=ESPO_PORTAL_ID:{PORTAL_ID}]
</IfModule>
```
...

**Expected behavior**
I expect the portal login page to load currently and it does not. The scripts calling /api/v1/portal-access/ result in a 501 Not Implemented error
**Screenshots**

**EspoCRM version**
Specify the EspoCRM version (or versions) on which the bug occurs.
7.4.3, 7.4.4
**Additional context**
It is unclear as to why the /client/ Alias works as expected as all the requests for those files seem to load fine. It is specific to the XHR requests for 
https://{custom-domain-name}/api/v1/portal-access/{portal-id}/Settings
https://{custom-domain-name}/api/v1/portal-access/{portal-id}/I18n?default=true

These both result in 501 Not Implemented errors with nothing in the Espo Log to debug the issue.

**Comment by yurikuzn** [comment]
This is not an Espo bug. Maybe needs fixing in docs, not sure.

**Comment by yurikuzn** [comment]
Please, such posts to be posted on the Forum > Installation and Upgrade Help.

**Comment by czcpf** [comment]
ModSecurity is set to not process any rules. I will repost in the forum. I posted in the developer Help section but have not received a response regarding v7. I really need some assistance in troubleshooting this.

**Comment by yurikuzn** [comment]
I'm the only who is on GitHub issues. I have a lot of work every day. We don't provide support on GitHub. I already spend too much time providing help and on false bug reports. Constantly postponing planned work. Request for helps reported as bugs really does not make my life better.

**Comment by czcpf** [comment]
I understand. I have contributed to the forum as well as reported other bugs here on GitHub which you have resolved. I have posted this issue on the forums as well and have received no support regarding this issue with v7 instructions.  I’m loosing a lot of time and money by not being able to resolve this after three days of troubleshooting. I would be happy to pay you for assistance. On May 12, 2023, at 9:44 AM, Yuri Kuznetsov ***@***.***> wrote:﻿
I'm the only who is on GitHub issues. I have a lot of work every day. We don't provide support on GitHub. I already spend too much time providing help and on false bug reports. Constantly postponing planned work. Request for helps reported as bugs really does not make my life better.
